### PR TITLE
Add missing @return tag in function comment

### PR DESCRIPTION
**Pull request by heykarthikwithu**

### CODE REVIEW

File: src/Entity/Form/AppCreateForm.php [hunk 1]
```diff
@@ -110,6 +110,10 @@ protected function alterFormWithApiProductElement(array &$form, FormStateInterfa
    * @param \Drupal\Core\Form\FormStateInterface $form_state
    *   The form state object.
    *
+   * @return array $elements
```
Comment (mxr576) [inline src/Entity/Form/AppCreateForm.php]: Weird that PHPCS did not complain about this in tests, it should have. But if you add the `@return` tag, always describe for humans what a function/method returns. Here it contains the "The API product render element."

**Review by mxr576 — CHANGES_REQUESTED** [review]

**Comment by mxr576** [comment]
https://travis-ci.org/apigee/apigee-edge-drupal/jobs/506176707 tests are failing

File: src/Entity/Form/AppCreateForm.php (hunk 1 above)
Comment (heykarthikwithu) [reply src/Entity/Form/AppCreateForm.php]: Added desription to the @return tag in function comment

**Comment by heykarthikwithu** [comment]
@mxr576 fixed now..

**Review by mxr576 — APPROVED** [review]

**Comment by mxr576** [comment]
Thanks!

**Pull request merged**